doc/telescope.txt

```diff
@@ -213,6 +213,154 @@ actions.toggle_selection({prompt_bufnr})          *actions.toggle_selection()*
 
 
 
+================================================================================
+                                                          *telescope.previewers*
+
+A collection of builtin previewers for telescope.
+
+Meant for different kind of pickers and should work universal given that
+the entries provide the fields the previewer expects.
+
+There is one basic previewer interface called Previewer, so has to be based
```

Comment:
> so has to be based

I'm kinda confused here, *what* has to be based on the Previewer interface?

Reply:
Its not really meant for other eyes. I just written it down and pushed because i had to run and wanted to see the docgen output before that.

But thanks for pointing it out. I gonna work more on it. I can ping you if its ready for review, if you wanna do one and when you are interested in how things work :)

Reply:
ah, I see. Alright then! :D

Reply:
Thanks for pointing that out. I have rewritten it :)